### ISSUE TITLE
Customize products and playlists available per site basis

### ISSUE THREAD

**Issue by naveen8408** [pnp/custom-learning-office-365#326]
Once we deploy the custom learning solution to the tenant is it possible to customize which products and playlists can appear for each site. The reason i'm asking is we have different agencies within our tenant who have different licenses within Office365 and we want to make sure they can only see products they have license to.

**Comment by juliemturner** [comment]
The guidance I'm personally providing to solve this issue is to add a second custom content pack for each additional agency you need to support.  Normally you would need to create the contents of said custom content pack but I'm going to suggest you just point again at the same CDN endpoint that the default content uses that way you can make separate customizations to it. Then do one of two things, either create a separate site collection to act as a learning portal for each of those agencies or embed the web part wherever needed for those agencies and then pin the web part to the agencies content pack. For more details on custom content packs you can see the description in this [video](https://www.youtube.com/watch?v=-sLXl_rKGxQ) (it's towards the end). In that I'm showing how to make your own, but you can also just use the base url of `https://pnp.github.io/custom-learning-office-365/learningpathways/`

**Comment by naveen8408** [comment]
But can agencies who have site owner or admin access on the site will have access to all content packs and can update content pack from webpart? If i create a separate learning portal for each of those agencies i'll be able to restrict availability of content packs within webpart?

**Comment by juliemturner** [comment]
Yes there is no security around content packs... you will have to have a level of trust with an "admin" for each agency to have access to the main learning pathways web site to customize their content set.

**Comment by naveen8408** [comment]
okay it would be great if we can audience target product tiles and playlists so that we can restrict them by ad groups.

**Comment by juliemturner** [comment]
I've marked it an enhancement request.

**Comment by juliemturner** [comment]
At this point, we won't be funding an enhancement for this issue due to our current backlog prioritization. That being said, we have tagged this Enhancement Request as a Contribution Opportunity in case community contributors would like to engage and help us all address the ask.

If anyone is interested in working on this enhancement, please reach out to discuss possible solutions so that we maintain consistency with the goals and functionality of the solution.
